pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage, lib, fetchFromGitHub, libcint, libxc, xcfun, blas,
+  numpy, scipy, h5py, python3
+} :
+
+buildPythonPackage rec {
+  pname = "pyscf";
+  version = "1.7.6.post1";
+
+  src = fetchFromGitHub {
+    owner = "pyscf";
+    repo = pname;
+    rev = "f6c9c6654dd9609c5e467a1edd5c2c076f793acc";
+    sha256  = "0xbwkjxxysfpqz72qn6n4a0zr2h6sprbcal8j7kzymh7swjy117w";
+  };
+
+  buildInputs = [
+    libcint
+    libxc
+    xcfun
+    blas
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    h5py
+    python3
+  ];
+
+  PYSCF_INC_DIR="${libcint}:${libxc}:${xcfun}";
+
+  doCheck = true;
```

Comment:
checkInputs and pythonImportsCheck missing.

Reply:
checkInputs is probably empty. I can't run any of the tests on my machine unfortunately. As mentioned above I have probably some file system issues. But at least it doesn't use pytest or anything as far as I can see.

Reply:
Disabled the tests again and therefore removed `checkInputs` as suggested by @markuskowa below. Keeping `pythonImportsCheck`.